google/cloud/bigtable/data/_metrics/handlers/opentelemetry.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import annotations
+
+from uuid import uuid4
+
+from google.cloud.bigtable import __version__ as bigtable_version
+from google.cloud.bigtable.data._metrics.handlers._base import MetricsHandler
+from google.cloud.bigtable.data._metrics.data_model import OperationType
+from google.cloud.bigtable.data._metrics.data_model import DEFAULT_CLUSTER_ID
+from google.cloud.bigtable.data._metrics.data_model import DEFAULT_ZONE
+from google.cloud.bigtable.data._metrics.data_model import ActiveOperationMetric
+from google.cloud.bigtable.data._metrics.data_model import CompletedAttemptMetric
+from google.cloud.bigtable.data._metrics.data_model import CompletedOperationMetric
+
+
+class _OpenTelemetryInstruments:
+    """
+    class that holds OpenTelelmetry instrument objects
+    """
+
+    def __init__(self, meter_provider=None):
+        if meter_provider is None:
+            # use global meter provider
+            from opentelemetry import metrics
+
+            meter_provider = metrics
+        # grab meter for this module
+        meter = meter_provider.get_meter("bigtable.googleapis.com")
+        # create instruments
+        self.operation_latencies = meter.create_histogram(
```

Comment:
let's make sure the metric descriptions are identical to our public doc https://cloud.google.com/bigtable/docs/client-side-metrics-descriptions

Reply:
fixed, although I didn't include this part of the client blocking latencies, since it doesn't apply: `For versions 2.21.0 and later, this metric also includes the latencies of requests queued on gRPC channels.`